### DISCUSSION TITLE
[Discussion] One takeover file was better than splitting each one separate

### DISCUSSION THREAD

**Discussion by tarunkant** [projectdiscovery/nuclei-templates#1176] [Q&A] [answered]
Hello,

I think the old version of the takeover file was better than splitting separately in this commit(https://github.com/projectdiscovery/nuclei-templates/commit/f6908465dc87b6b1aa707c026ad1d79a001bf636).

**Why do I think that?**
Now to test takeover on one domain, the nuclei will make separate requests for all templates. But earlier one request could give us the final result with testing of all takeovers.

**What can be done?**
- Changing to that folder with the older version of it
- Add the older file too, so that the user who wants to test all takeovers in one shot can use that.

Please share your views of what you think of it.

Thanks,

**Comment by ehsandeep** [comment, marked as answer]
Hi @tarunkant, 

Check the similar issue on the same https://github.com/projectdiscovery/nuclei-templates/issues/1145

> the nuclei will make separate requests for all templates. 

No, it does not as detailed in the referenced issue.

and now we can add more strict matchers to each template instead of having limitations of following OR conditions with the previous format.
